lib/mayaUsd/ufe/Global.cpp

```diff
@@ -107,6 +107,12 @@ StagesSubject::Ptr g_StagesSubject;
 bool InPathChange::inGuard = false;
 bool InAddOrDeleteOperation::inGuard = false;
 
+void refreshStages()
```

Comment:
I'm surprised that this would need to be explicitly called.  This clears out USD observation, and causes the stage map to be rebuilt.  When is this needed outside of file open?